canonical/canonical.com
# Apply redesign /partners/find-a-partner and add new /silicon partner page

## Done

- Applied redesign as per [mock-up](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9305-22310&t=9u07xXTxkSaNqaby-0)
- Added new Silicon page to partners bubble as per [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9614%3A20510&t=cuBOBVZg6HBHGplX-1) 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-894.demos.haus/partners/find-a-partner
- Check page against desgin
- Check filters and see that filter count is updated in the following ways:
   - If no filters are applied the text should read "All X partners"
   - If only one partner exists the text should read "X partner" (check this by typing "qualc" in the search bar, only QualComm should appear)
   - If there are no partners the text should read "0 partners" and there should be a notification prompting the user to search again
- Check filter state is maintained on refresh
- Check that on small screens the "Toggle filters" and "Clear filters" buttons show up and work as expected
  - Toggle filters should show/hide available filters
  - Clear filters should clear all selected filters
- Check that "Silicon" is a programme filter option (should yield 14 partners)
- Go to https://canonical-com-894.demos.haus/partners/silicon
  - See that page matches design  

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3746, https://warthogs.atlassian.net/browse/WD-4330
